src/sentry/static/sentry/app/views/projectAlertSettings.jsx

```diff
@@ -214,14 +214,6 @@ const GeneralSettings = React.createClass({
                 {t('Unable to save your changes. Please ensure all fields are valid and try again.')}
               </div>
             }
-            <TextField
-                key="subjectPrefix"
-                label={t('Subject prefix')}
-                value={formData.subjectPrefix}
-                required={false}
-                error={errors.subjectPrefix}
-                onChange={this.onFieldChange.bind(this, 'subjectPrefix')}
-                help="Sentry will prefix all emails for this project with the given value." />
```

Comment:
I removed this field since it's already included on the general project settings, and it's not limited to alerts.